### PR TITLE
Handle 'version' and 'latest_version' in item table

### PR DESCRIPTION
**Pull request by stephanheilner**
I'm not sure how to test this, but it looks right to me. My python is not mucho bueno.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Typical usage looks like this:
     
     item = catalog.item(uri="/scriptures/bofm", lang="eng")
     
-    item_package = ItemPackage(item_external_id=item['external_id'], item_version=item['latest_version'])
+    item_package = ItemPackage(item_external_id=item['external_id'], item_version=item['version'])
```
Comment (hiltonc) [inline README.md]: Is this true for all v3 catalogs now that it was "rebooted"?

File: gospellibrary/item_packages.py [hunk 2]
```diff
@@ -82,15 +82,26 @@ def html(self, uri=None, subitem_uri=None, paragraph_id=None):
         with sqlite3.connect(item_package_path) as db:
             c = db.cursor()
             try:
-                if uri:
-                    c.execute('''SELECT content, start_index, end_index FROM subitem_content_range
-                                     INNER JOIN subitem_content ON subitem_content_range.subitem_id=subitem_content.subitem_id
-                                 WHERE uri=?''', [uri])
+                if self.schema_version == '2.0.3':
```
Comment (hiltonc) [inline gospellibrary/item_packages.py]: I think this is just a table rename. If so, can you do a string substitution? In Python that would be something like:

    '''blah {my_table} blah'''.format(my_table='table_name')

File: README.md (hunk 1 above)
Comment (stephanheilner) [inline README.md]: Yes, this python will always use the column name "version" now, even with 2.0.3 content

File: gospellibrary/item_packages.py (hunk 2 above)
Comment (stephanheilner) [inline gospellibrary/item_packages.py]: Love it!

File: README.md (hunk 1 above)
Comment (stephanheilner) [inline README.md]: yes

**Pull request merged**